src/addrman.h

```diff
@@ -593,6 +577,8 @@ class CAddrMan
         Check();
     }
 
+    const std::vector<bool>& GetAsmap() { return m_asmap; }
```

Comment:
4ffc8b81bc520fe66fb is there any benefit to making `m_asmap` private rather than a public data member? referencing our previous conversation:

- https://github.com/bitcoin/bitcoin/pull/20210#issuecomment-766093925

- https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#c131-avoid-trivial-getters-and-setters

(otherwise, make the getter const?)
```suggestion
    const std::vector<bool>& GetAsmap() const { return m_asmap; }
```

Reply:
I've made the getter const. I think removing the only public data member of CAddrMan is an improvement.